helpers/helpers.go

```diff
@@ -294,6 +294,7 @@ func RunCommand(cmdname string, args ...string) error {
 	cmd := exec.Command(cmdname, args...)
 	cmd.Stdout = os.Stdout
 	cmd.Stderr = os.Stderr
+	cmd.Stdin = os.Stdin
```

Comment:
This is added to facilitate launching applications that read input (i.e., the editors). I considered adding the same thing to the `RunCommandOutput` function below, but didn't know how people would want it. Perhaps a separate `RunCommandInput` function that takes a `bytes.Buffer` as input? We can add that if we have a need.

I made sure the `helpers_tests` still ran as expected after adding this line.

Reply:
Great idea. I'll redo this.